karma.conf.js

```diff
@@ -1,8 +1,8 @@
-// Karma configuration
-// Generated on Tue Jan 24 2017 12:22:50 GMT-0500 (EST)
+// Karma configuration file, see link for more information
+// https://karma-runner.github.io/0.13/config/configuration-file.html
```

Comment:
It is boilerplate code I copy and pasted, so I cannot take credit for it, but I agree!